src/QuadForm/Herm/Genus.jl

```diff
@@ -1633,10 +1633,20 @@ end
 #
 ################################################################################
 
-# Return b, p, bad
-# b = isdefinite(L)
-# p = prime ideal of base_ring(L) which can be used for the neighbor method
-# bad = bad primes of L, where L,p is not modular or p is dyadic and dividing discriminant(S)
+
+@doc Markdown.doc"""
+    smallest_neighbour_prime(L::HermLat) -> Bool, NfRelOrdIdl, Vector{NfOrdIdl}
+
+Given a hermitian lattice `L`, return `def, P0, bad` such that:
+
+- `def` is `true` if `L` is definite, else `false`;
+- `P0` is a prime ideal in the base ring `O` of `L` which is not bad, such that
+  `L` is isotropic at `minimum(P0)` and `P0` has smallest minimum among the primes 
+  satisfying these properties;
```

Comment:
```suggestion
  satisfying these properties; if `L` is indefinite, `P0` is set to be the trivial ideal
```

Reply:
(Or we say it is any ideal in the indefinite case and don't test it.)

I have no strong preference.